apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -66,6 +66,13 @@ const SummaryResponses = ({
     [scriptData, selectedSection]
   );
 
+  const eventData = {
```

Comment:
I did initially try getting this data all in redux in the FreeResponseResponses component but it got quite messy - it seemed like it might need to come from a few different redux stores.  I chose this strategy instead as all the data we needed was available in the scriptData.   I am still getting the hang of "when to pass through data vs when to get it from redux" so any additional considerations / reasoning would be appreciated.